### PR TITLE
Fix android pagination scale and fix ios bugs

### PR DESCRIPTION
**Pull request by ghost**

### CODE REVIEW

File: packages/flask/src/UICarouselView/CarouselView/Pagination.tsx [hunk 1]
```diff
@@ -17,9 +23,18 @@ function Circle({ active, onPress, theme }: CircleProps) {
     const { animatedStyles } = usePaginationStyle(active, theme);
 
     return (
-        <TouchableOpacity hitSlop={UIConstant.carousel.circleHitSlop} onPress={onPress}>
-            <Animated.View style={[styles.circle, animatedStyles]} />
-        </TouchableOpacity>
+        <Animated.View style={[animatedStyles, styles.circle]}>
+            <TouchableOpacity hitSlop={UIConstant.carousel.circleHitSlop} onPress={onPress}>
+                <UIBackgroundView
```
Comment (savelichalex) [inline packages/flask/src/UICarouselView/CarouselView/Pagination.tsx:27]: Maybe make it also Animated? I guess there is no need to remove animation completely, just separate it to two components

Comment (ghost) [reply packages/flask/src/UICarouselView/CarouselView/Pagination.tsx:27]: @savelichalex it happens so quickly that I can't see the difference with and without color animation. Do we really need animation?

Comment (savelichalex) [reply packages/flask/src/UICarouselView/CarouselView/Pagination.tsx:27]: It's up to you, as soon as it looks good

File: packages/flask/src/UICarouselView/CarouselView/CarouselContainer/index.native.tsx [hunk 2]
```diff
@@ -1,24 +1,31 @@
...
 
     return pages.map((page, index) => {
-        const onPressPage = () => onPress(index);
+        const onPressPage = () => {
+            if (Platform.OS === 'android') {
+                nextPage(index);
+            } else {
+                onPress(index);
+            }
+        };
```
Comment (savelichalex) [inline packages/flask/src/UICarouselView/CarouselView/CarouselContainer/index.native.tsx]:
```suggestion
        const onPressPage = Platform.select({ android: nextPage, default: onPress });
```

That way you won't create additional wrapper function :)

File: packages/flask/src/UICarouselView/CarouselView/CarouselContainer/index.web.tsx [hunk 3]
```diff
@@ -156,6 +176,7 @@ export function CarouselViewContainer({
                     );
                 })}
             </Animated.View>
+            {renderPagination()}
```
Comment (savelichalex) [inline packages/flask/src/UICarouselView/CarouselView/CarouselContainer/index.web.tsx]:
```suggestion
            {showPagination && (
                 <Pagination
                     pages={pages}
                     activeIndex={currentIndexRef.current}
                     setPage={jumpToIndex}
                 />
             )}
```

Let's simply inline it

File: packages/flask/src/UICarouselView/CarouselView/CarouselContainer/index.native.tsx [hunk 4]
```diff
@@ -99,6 +117,7 @@ export function CarouselViewContainer({
             >
                 {returnPages(pages, shouldPageMoveOnPress, nextPage)}
             </AnimatedPagerView>
+            {renderPagination()}
```
Comment (savelichalex) [inline packages/flask/src/UICarouselView/CarouselView/CarouselContainer/index.native.tsx]:
```suggestion
            {showPagination && (
                 <Pagination pages={pages} activeIndex={currentIndex} setPage={setPage} />
             )}
```

Same, let's just inline it :)

**Review by savelichalex — APPROVED** [review]

**Review by vetkamak — APPROVED** [review]

**Pull request merged**